src/components/molecules/GenericBarChartView.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react';
+import { ResponsiveContainer, BarChart, LabelList, XAxis, Bar , Tooltip, Legend} from 'recharts';
+import { roseColor, honeyColor, yellowColor, blackColor, whiteColor } from 'style';
+import { Typography } from 'components/atoms';
+import tinycolor from 'tinycolor2';
+
+
+interface IProps {
+  isStacked: boolean;
```

Comment:
can be enum:
```
barType {
  single, //(default)
  multi,
  stacked
}
```

Reply:
will do, but I think it might complicate things. will consult.

**single** 
_numOfBars_:1
_stacked_: no
**multi**
_numOfBars_: 2-X
_Stacked_: yes/no
**stacked**
_numOfBars_: 2-X
_Stacked_: yes